site/content/docs/5.0/utilities/inset.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Inset
+description: Use inset utilities to quickly style position of an element.
+group: utilities
+toc: true
+---
+
+## Inset
+
+Use inset utilities to position elements. You can combine it with position and display utilities
+
+{{< example class="bd-example-inset-utils">}}
```

Comment:
Not mandatory at all but by using `justify-content-between` utility here you might drop you `margin`s on `div`. :)

Reply:
Okasss, I can do that